java/mapreduce/src/test/org/apache/orc/mapred/TestOrcFileEvolution.java

```diff
@@ -312,6 +314,26 @@ public void testPositional2() {
         false, addSarg, true);
   }
 
+  private void checkEvolutionPosn(String writerType, String readerType,
```

Comment:
This function seems to be duplicated mostly with `checkEvolution`. Is the only difference is the column name, `_col0` and `a`?

Reply:
Ya. I also don't have a better option. +1 for adding this.

Reply:
This was primarily to fix the errors in some of the tests that were using positional names